### PR TITLE
change_layout_login_page

### PR DESCRIPTION
**Pull request by arbins**

### CODE REVIEW

**Comment by arbins** [comment]
screen shot 2016-05-21 at 9 47 39 pm

File: app/views/devise/confirmations/new.html.haml [hunk 1]
```diff
@@ -1,4 +1,5 @@
-%h2 Resend confirmation instructions
+.page-header
+	%h2.text-uppercase.text-center Resend confirmation instructions
```
Comment (lemattma) [inline app/views/devise/confirmations/new.html.haml]: indentation? it should be 2 spaces

File: app/views/devise/sessions/new.html.haml [hunk 2]
```diff
@@ -1,13 +1,17 @@
...
-  .form-actions
-    = f.button :submit, 'Log in', class: 'btn btn-primary'
-  %br
-
-= render 'devise/shared/links'
+.col-md-4.col-md-offset-4
+  .header
+    .container-box
+      .heading.text-center
+        LOGIN IN
```
Comment (lemattma) [inline app/views/devise/sessions/new.html.haml]: Keep it consistent, write normal case text and use css for upcase.

**Comment by lemattma** [comment]
:+1:

**Pull request merged**